server/interceptor/interceptor.go

```diff
@@ -401,9 +401,11 @@ func newRPCInfo(req interface{}, quotaUser string) (*rpcInfo, error) {
 			{Group: quota.Global, Kind: kind},
 		}
 		switch req := req.(type) {
-		case logLeavesRequest:
+		case *trillian.QueueLeavesRequest:
 			info.tokens = len(req.GetLeaves())
-		case mapLeavesRequest:
+		case *trillian.AddSequencedLeavesRequest:
+			info.tokens = len(req.GetLeaves())
+		case *trillian.SetMapLeavesRequest:
 			info.tokens = len(req.GetLeaves())
 		default:
 			info.tokens = 1
```

Comment:
Would if be excessively onerous to list all request types here, and error if one is encountered that isn't handled? That'll stop anyone forgetting to update this list.

Reply:
Oh, the yaks. 

I've gone one further and merged this type switch with the bigger type switch in the function above.  A bunch of boilerplate like that needs careful review, but hopefully should be less brittle to subtle changes in the future.